azure.yaml

```diff
@@ -2,7 +2,19 @@ name: contoso-chat-csharp-prompty
 metadata:
   template: contoso-chat-csharp-prompty@0.0.1
 services:
-  ContosoChatAPI:
-    project: ./src/ContosoChatAPI
-    image: "${AZURE_CONTAINER_REGISTRY_NAME}.azurecr.io/contosochatapi:latest"
-    host: containerapp
\ No newline at end of file
+  api:
+    host: containerapp
+    language: csharp
+    project: src/ContosoChatAPI/ContosoChatAPI
+hooks:
+  postprovision:
+    posix:
+      shell: sh
+      continueOnError: false
+      interactive: true
+      run: infra/hooks/postprovision.sh
+    windows:
+      shell: pwsh
+      continueOnError: false
+      interactive: true
+      run: infra/hooks/postprovision.ps1
```

Comment:
Returning an error code 126 for azd up due to chmod. I will push a fix with another change

Reply:
I opened a PR against your fork's branch. I still am facing errors on `azd deploy` related to the api service that have nothing to do with this PR